contract/signals.py

```diff
@@ -120,6 +125,30 @@ def activate_contracted_policies(sender, instance, **kwargs):
                         __save_or_update_contract(contract, contract.user_updated)
 
 
+# additional filters for payment in 'contract' tab
+def append_contract_filter(sender, **kwargs):
+    user = kwargs.get("user", None)
+    additional_filter = kwargs.get('additional_filter', None)
+    if "contract" in additional_filter:
```

Comment:
just check with @rstencel  that he will send "contract" and not "Contract"

Reply:
ok. I will ask him about it

Reply:
@delcroip Ralal has  just confirmed me that "contract". Therefore I don't have to change this.